sql-join/magic-mallrats.sql

```diff
@@ -0,0 +1,6 @@
+SELECT "customers"."firstName",
+       "customers"."lastName"
+  FROM "customers"
+  JOIN "rentals" USING ('customerId')
+  JOIN "inventory" USING ('inventoryId')
+  WHERE "filmId" = (547)
```

Comment:
Don't use `()` around the number, because that is a different data type. Postgres apparently converts it for you, though. But it's best not to use it.

Reply:
The column names in your "USING" clause need to use double quotes.